src/components/Comment/NestedComment/NestedComment.jsx

```diff
@@ -17,124 +14,89 @@ export default function NestedComment({
   isFirst,
   onCommentOptionClick,
 }) {
-  const { setIsEdit, commentId, setCommentId, setContent, inputFocus } =
-    useCommentContext();
-  const { deleteComment } = useComment();
-  const [isOptionModalOpen, setIsOptionModalOpen] = useState(false);
-  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+  const { setCommentId } = useCommentContext();
 
   const { like, deleteLike } = useLike({ type: 'comments', typeId: data.id });
 
-  const onCloseClick = () => {
-    setCommentId(undefined);
-    setContent('');
-  };
-
   const {
-    id,
-    postId,
     userDisplay,
-    isWriter,
     isWriterWithdrawn,
     content,
     isLiked,
     likeCount,
     createdAt,
-    updatedAt,
     isVisible,
     isUpdated,
     isDeleted,
     // isLiked,
-    children,
   } = data;
 
   return (
-    <>
-      <div
-        className={styles.nestedComment}
-        onClick={(e) => e.stopPropagation()}
-        style={{
-          paddingBottom: isLast ? '6px' : 'none',
-          borderBottomLeftRadius: isLast ? '5px' : '0px',
-          borderBottomRightRadius: isLast ? '5px' : '0px',
-        }}
-      >
-        <div className={styles.nestedCommentTop}>
-          <div className={styles.commentTopLeft}>
-            <div className={styles.nestedIcon}>
-              {isFirst && <Icon id='nested-arrow' width='15' height='15' />}
-            </div>
-            <div className={styles.cloud}>
-              <Icon id='cloud' width='19' heigth='13' />
-            </div>
-            <p className={`${isWriterWithdrawn && styles.isWriterWithdrawn}`}>
-              {isWriterWithdrawn ? '(알 수 없음)' : userDisplay}
-            </p>
-            <p className={styles.dot}>·</p>
-            <p>
-              {timeAgo(createdAt)} {isUpdated ? ' (수정됨)' : null}
-            </p>
+    <div
+      className={styles.nestedComment}
+      onClick={(e) => {
+        e.stopPropagation();
+        setCommentId(undefined);
+      }}
+      style={{
+        paddingBottom: isLast ? '6px' : 'none',
+        borderBottomLeftRadius: isLast ? '5px' : '0px',
+        borderBottomRightRadius: isLast ? '5px' : '0px',
+      }}
```

Comment:
인라인 스타일은 가급적 지양하면 좋을 것 같아요~!

스타일 관련 속성은 스타일 파일에서만 관리하는 것이 유지보수 측면에서 좋을 것 같아요 :)
그리고 인라인 스타일은 브라우저 성능상 좋지 않다고 합니다 🥲

`none > undefined`
`0px > 0`으로 대체할 수 있습니다

```suggestion
      style={{
        paddingBottom: isLast ? '6px' : undefined,
        borderBottomLeftRadius: isLast ? '5px' : '0',
        borderBottomRightRadius: isLast ? '5px' : '0',
      }}
```